src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,32 @@
+import styles from "./footer.module.scss";
+import GithubIcon from "./GithubIcon";
+
+const Footer = () => {
+	return (
+		<footer className={styles.footer}>
+			<p>
+				Made with ♥ by{" "}
+				<a target="_blank" href="https://github.com/yashchaudhari008">
+					yashchaudhari008
+				</a>{" "}
+				&{" "}
+				<a
+					target="_blank"
+					href="https://github.com/yashchaudhari008/minime/graphs/contributors"
+				>
+					contributors
+				</a>
+				<span className={styles.stick}> | </span>
+				<a
+					target="_blank"
+					className={styles.github}
+					href="https://github.com/yashchaudhari008/minime"
+				>
+					<GithubIcon /> Github Page
```

Comment:
Use `<FontAwesomeIcon icon={faGithub} className={styles.icon} />` . You might need to add `@fortawesome/free-brands-svg-icons` to add `faGithub` icon

References:
- https://fontawesome.com/docs/web/use-with/react/add-icons#add-individual-icons-explicitly
- You can also look into `src/components/widgets/AddNewWidgetBtn/AddNewWidgetBtn.tsx` to get idea to implement